docs/HowTo/Interact/APIs/Engine-API.md

```diff
@@ -9,20 +9,19 @@ These [API methods](../../../Reference/Engine-API-Methods.md) are a separate sub
 
 ## Configure the Engine API
 
-To configure the Engine API, specify the [service ports](#service-ports) and [host allowlist](#host-allowlist).
-Also [enable the JSON-RPC](API.md#enabling-api-access).
-Ensure the [`ETH` method is enabled](Using-JSON-RPC-API.md#api-methods-enabled-by-default) (it's enabled by default).
+To configure the Engine API:
 
-### Service ports
-
-To specify the port the Engine API service listens on, use the
-[`--engine-rpc-http-port`](../../../Reference/CLI/CLI-Syntax.md#engine-rpc-http-port) or
-[`--engine-rpc-ws-port`](../../../Reference/CLI/CLI-Syntax.md#engine-rpc-ws-port) options.
+- Enable the Engine API using the [`--engine-rpc-enabled`](../../../Reference/CLI/CLI-Syntax.md#engine-rpc-enabled) option.
```

Comment:
This is actually deprecated in the latest besu releases, and we instead defer to the presence or absence of `terminalTotalDifficulty` in the genesis config.  Relevant changes:

[engine-rpc-enabled parameter deprecated](https://github.com/hyperledger/besu/blob/main/besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java#L569-L574)

[log warning if engine-rpc-enabled used](https://github.com/hyperledger/besu/blob/main/besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java#L2115-L2119)

[merge config and therefore engine api deferring to genesis](https://github.com/hyperledger/besu/blob/main/besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java#L3192-L3201)